src/openxr/OpenXRApi.cpp

```diff
@@ -51,15 +52,293 @@ XRAPI_ATTR XrResult XRAPI_CALL xrLocateHandJointsEXT(
+														   "			{\n"
+														   "				\"set\": \"godot\",\n"
+														   "				\"action\": \"joystick\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/thumbstick\",\n"
+														   "					\"/user/hand/right/input/thumbstick\"\n"
+														   "				]\n"
+														   "			},\n"
+														   "			{\n"
+														   "				\"set\": \"godot\",\n"
+														   "				\"action\": \"ax_buttons\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/x/click\",\n"
+														   "					\"/user/hand/right/input/a/click\"\n"
+														   "				]\n"
+														   "			},\n"
+														   "			{\n"
+														   "				\"set\": \"godot\",\n"
+														   "				\"action\": \"bym_button\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/y/click\",\n"
+														   "					\"/user/hand/right/input/b/click\"\n"
+														   "				]\n"
+														   "			},\n"
+														   "			{\n"
+														   "				\"set\": \"godot\",\n"
+														   "				\"action\": \"front_button\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/trigger/value\",\n"
+														   "					\"/user/hand/right/input/trigger/value\"\n"
+														   "				]\n"
+														   "			},\n"
+														   "			{\n"
+														   "				\"set\": \"godot\",\n"
+														   "				\"action\": \"side_button\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/squeeze/value\",\n"
+														   "					\"/user/hand/right/input/squeeze/value\"\n"
+														   "				]\n"
+														   "			},\n"
+														   "		],\n"
+														   "	},\n"
+														   "	{\n"
+														   "		\"path\": \"/interaction_profiles/valve/index_controller\",\n"
+														   "		\"bindings\": [\n"
+														   "			{\n"
+														   "				\"set\": \"godot\",\n"
+														   "				\"action\": \"aim_pose\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/aim/pose\",\n"
+														   "					\"/user/hand/right/input/aim/pose\"\n"
+														   "				]\n"
+														   "			},\n"
+														   "			{\n"
+														   "				\"set\": \"godot\",\n"
+														   "				\"action\": \"front_trigger\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/trigger/value\",\n"
+														   "					\"/user/hand/right/input/trigger/value\"\n"
+														   "				]\n"
+														   "			},\n"
+														   "			{\n"
+														   "				\"set\": \"godot\",\n"
+														   "				\"action\": \"side_trigger\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/squeeze/value\",\n"
+														   "					\"/user/hand/right/input/squeeze/value\"\n"
```

Comment:
I've added the bindings, not sure if it's enough. I've added this more because it is functionality the other plugins offer as well. Also in godot-xr-tools I use this input to animate the bottom 3 fingers of a hand model.
I agree that the button input is far more useful for the grip.